src/integrations/prefect-kubernetes/prefect_kubernetes/worker.py

```diff
@@ -636,6 +637,10 @@ async def _get_configured_kubernetes_client(
         Returns a configured Kubernetes client.
         """
         client = None
+        if os.environ.get(
+            "PREFECT_KUBERNETES_WORKER_ADD_TCP_KEEPALIVE", "TRUE"
```

Comment:
why does this look up an environment variable instead of a setting?

Reply:
Mainly to maintain the same api as before and because we configure other settings with env vars like `PREFECT_KUBERNETES_CLUSTER_UID`. Should we move those to settings also?

Reply:
At some point I think we should, but it's not necessary to do in this PR - I was mainly just curious